modules/math/src/main/java/com/opengamma/strata/math/impl/interpolation/Interpolator2D.java

```diff
@@ -13,7 +13,7 @@
 /**
  * A base class for two-dimensional interpolation.
  */
-public abstract class Interpolator2D implements Interpolator<Map<Double, Interpolator1DDataBundle>, DoublesPair> {
+public abstract class Interpolator2D {
```

Comment:
If the `Interpolator1D` actually needs to be serialisable then presumably this does too.